src/components/input/input.scss

```diff
@@ -254,26 +278,11 @@ md-input-container {
     .md-auto-hide {
       .md-input-message-animation {
         opacity: 0;
-        margin-top: -100px;
+        margin-top: -1em;
```

Comment:
What if the message is larger than `1em`? Won't this just "jump" in part-way through?